pkg/osquery/tables/dev_table_tooling/commands_darwin.go

```diff
@@ -0,0 +1,14 @@
+//go:build darwin
+// +build darwin
+
+package dev_table_tooling
+
+var allowedCommands = map[string]allowedCommand{
+	"echo": {
+		binPaths: []string{"echo"},
+		args:     []string{"hello"},
+	},
+}
+
+// Platform-specific test data
+var echoHelloOutput = "hello\n"
```

Comment:
Feels a little odd to me to have a variable only used for testing outside of a `_test.go` file. I would declare a var in `table_test.go` as a map of os -> output inside the `Test_generate()` like:

```
expectedOutputs := map[string]string{
  "windows": "hello\n",
  "darwin": "hello\n",
  "linux": "hello\r\n",
}
```

then access via

```
expectedOutput := expectedOutputs[runtime.GOOS]
```

inside the test case

Reply:
Yeah I didn't like the messiness of my first approach - your suggestion is definitely an improvement.

I reworked the test to avoid needing to know about newline characters in [5cc1059e](https://github.com/kolide/launcher/pull/881/commits/5cc1059e4dc0e29ace806ffed8719c8fa6a8589c)

It does add some complexity to the test verification code, but makes the tests easier to specify and maintain.